hack/run-ci-e2e-test.sh

```diff
@@ -46,15 +42,43 @@ NODE_COUNT=${NODE_COUNT:-2}
 SKIP_NODE_DELETION=${SKIP_NODE_DELETION:-"-skip-node-deletion=false"}
 KEY_PAIR_NAME=${KEY_PAIR_NAME:-"libra"}
 
+# OPERATOR_IMAGE defines where the WMCO image to test with is located. If $OPERATOR_IMAGE is already set, use its value.
+# In OpenShift CI $IMAGE_FORMAT will be set to a value such as registry.svc.ci.openshift.org/ci-op-<input-hash>/stable:${component}
+# We need to replace ${component} with the name of the WMCO image.
+# The stable namespace is used as it will use the last promoted version when the test is run in an repository other than this one.
+# When running in this repository the image built in the pipeline will be used instead.
+# https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help/ci-operator#release
+OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
+
+MANIFEST_LOC="/tmp/ci-wmco-manifest"
+cp -r deploy/olm-catalog/windows-machine-config-operator/manifests $MANIFEST_LOC
+sed -i "s|REPLACE_IMAGE|$OPERATOR_IMAGE|g" $MANIFEST_LOC/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+
 cd $WMCO_ROOT
 oc create -f deploy/namespace.yaml
+oc create secret generic cloud-credentials --from-file=credentials=$AWS_SHARED_CREDENTIALS_FILE -n windows-machine-config-operator
+oc create secret generic cloud-private-key --from-file=private-key.pem=$KUBE_SSH_KEY_PATH -n windows-machine-config-operator
+oc create secret generic kubeconfig --from-file=kubeconfig=$KUBECONFIG -n windows-machine-config-operator
+
+# Run the operator in the windows-machine-config-operator namespace
+# Currently this fails even on successes, adding this if to ignore the failure
+if ! operator-sdk run --olm --olm-namespace openshift-operator-lifecycle-manager --operator-namespace windows-machine-config-operator --operator-version 0.0.1 --manifests $MANIFEST_LOC; then
+  echo operator-sdk run failed
+fi
+
 # The bool flags in golang does not respect key value pattern. They follow -flag=x pattern.
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
 # Run the creation tests and skip deletion of the Windows VMs
-$OSDK test local ./test/e2e --debug --up-local --operator-namespace=windows-machine-config-operator --local-operator-flags "--zap-level=debug --zap-encoder=console" --go-test-flags "-run=TestWMCO/create -v -timeout=60m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME"
+$OSDK test local ./test/e2e --no-setup --debug --operator-namespace=windows-machine-config-operator --go-test-flags "-run=TestWMCO/create -v -timeout=60m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME"
```

Comment:
How is the logging now? Is it still showing debug level info in the operator?

Reply:
I don't think it should run with debug level,we'd need to add that option to the container's run command

Reply:
Please add that otherwise it will hard to debug CI issues.